getting-started/security_policy_specification.md

```diff
@@ -67,6 +67,26 @@ spec:
     - capability: [capability name]
       fromSource:                          # --> optional
       - path: [absolute exectuable path]
+  
+  syscalls:
+    matchSyscalls:
+    - syscall:
+      - syscallX
+      - syscallY
+      fromSource:                            # --> optional
+      - path: [absolute exectuable path]
+      - dir: [absolute directory path]
+        recursive: [true|false]              # --> optional
+    matchPaths:
+    - path: [absolute directory path | absolute exectuable path]
+      recursive: [true|false]                # --> optional
+      - syscall:
+        - syscallX
+        - syscallY
+      fromSource:                            # --> optional
+      - path: [absolute exectuable path]
+      - dir: [absolute directory path]
+        recursive: [true|false]              # --> optional
 
   action: [Allow|Audit|Block] (Block by default)
```

Comment:
We do not support action: allow/block for syscalls ... Also it is mentioned "Block by default". We certainly do not support that action too. 

Can we update the document to explain that syscall constructs are primarily from auditing perspective?